engine/src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -25,14 +25,14 @@ trait SwaggerService extends SwaggerUiResourceHttpService {
 }
 
 object CromwellApiServiceActor {
-  def props(workflowManagerActorRef: ActorRef, validateActorRef: ActorRef, config: Config): Props = {
-    Props(classOf[CromwellApiServiceActor], workflowManagerActorRef, validateActorRef, config)
+  def props(workflowManagerActorRef: ActorRef, config: Config): Props = {
+    Props(classOf[CromwellApiServiceActor], workflowManagerActorRef, config)
   }
 
   def traceName(name: String) = Monitor.traceName(name)
 }
 
-class CromwellApiServiceActor(val workflowManager: ActorRef, val workflowDescriptorMaterializer: ActorRef, config: Config)
+class CromwellApiServiceActor(val workflowManager: ActorRef, config: Config)
```

Comment:
NOTE: I almost missed removing this cruft, and why I prefer `override val` to just `val`, even if it's ugly. 😉